source/views/news/news-item.js

```diff
@@ -28,43 +28,18 @@ export default function NewsItem(props: {
 }) {
   const {story, embedFeaturedImage} = props.navigation.state.params
 
+  // Use local or remote styles
+  let newsCSS = ''
+  if (process.env.NODE_ENV === 'development') {
+    let css = require('../../../docs/news-styles.json')
+    newsCSS = `<Style>${css}</Style>`
```

Comment:
Lower-case `Style`. It's an html element.

Reply:
oops